dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFieldsTest.java

```diff
@@ -121,7 +126,82 @@ public void failWithNegativeContigCoordinates() {
     public void failWithNegativeChromosomeCoordinates() {
         new SubSnpCoreFields(12345, Orientation.FORWARD, 123L, Orientation.FORWARD, "contigName", 1L, 10L,
                              Orientation.REVERSE, LocusType.SNP, "chromosomeName", -5L, 50L, "T", "T", "A", "T/A", "",
-                             null, null, Orientation.FORWARD, "", null, null, Orientation.FORWARD, "batch");
+                             null, null, Orientation.FORWARD, "", null, null, Orientation.FORWARD, null, "batch");
+    }
+
+    @Test
+    public void testGenotypeList() {
```

Comment:
This method is testing 5 different cases so it would be better split in 5 for clarity.